### PR TITLE
Add: Verify page flag

### PR DESCRIPTION
**Pull request by andygout**
cc @rowanmanning @GlynnPhillips @lc512k 

Dependent on this `pa11y` PR being merged: https://github.com/pa11y/pa11y/pull/224

### CODE REVIEW

**Comment by rowanmanning** [comment]
We might not need this as a command-line flag, as you'll be able to set it in your config file. Would the flag be useful, considering not all pages will necessarily match the same regexp?

**Comment by andygout** [comment]
I was wondering that and what you suggest makes infinitely more sense - will change accordingly.

**Comment by andygout** [comment]
This is what test output would look like when urls fail on basis of not verifying specific content on the page.

screen shot 2016-12-14 at 17 46 59

File: README.md [hunk 1]
```diff
@@ -149,7 +150,7 @@ make test-integration    # Run the integration tests
 
 ## Licence
 
-Licensed under the [Lesser General Public License (LGPL-3.0)](LICENSE).
```
Comment (rowanmanning) [inline README.md]: Same whitespace issue as in https://github.com/pa11y/pa11y/pull/224#discussion_r92574725

**Pull request merged**